src/domain/header/_accessibilityMenu.scss

```
@@ -0,0 +1,33 @@
+.app-AccessibilityMenu {
+    padding: 15px 30px;
+    width: 420px;
+    overflow: scroll;
```

```suggestion
    overflow: hidden;
```

Otherwise the menu will have two scrollbars.

---

Good catch!